recipe/meta.yaml

```diff
@@ -62,6 +62,7 @@ requirements:
     - aom
     - svt-av1
     - ffnvcodec-headers  # [linux64 or win]
+    - libopus
```

Comment:
https://github.com/conda-forge/libopus-feedstock/ hasn't had a build in >2 years, but then, opus hasn't had a [release](https://gitlab.xiph.org/xiph/opus/-/tags) for even longer.

Reply:
Hmm, maybe somebody can rerender to make sure that recent compilers have not imposed new errors on bad C coding practices. Otherwise, should be fine.

But I understand your "maintainability" concern.

Reply:
I guess there is also a recent uptick in the number of commits https://gitlab.xiph.org/xiph/opus/-/commits/master

Reply:
Rerendering went fine. https://github.com/conda-forge/libopus-feedstock/pull/7